### PR TITLE
Add support for generation of validator keys

### PR DESCRIPTION
**Pull request by siy**

### CODE REVIEW

File: radixdlt-cli/build.gradle [hunk 1]
```diff
@@ -42,11 +42,12 @@ compileJava {
...
-    compile 'org.codehaus.groovy:groovy-all:2.4.15'
-    compile group: 'com.google.code.gson', name: 'gson', version: '2.8.2'
-    compile group: 'commons-cli', name: 'commons-cli', version: '1.4'
-    compile 'info.picocli:picocli:4.2.0'
-    annotationProcessor 'info.picocli:picocli-codegen:4.2.0'
+    compile 'org.codehaus.groovy:groovy-all:3.0.5'
+    compile 'com.google.code.gson:gson:2.8.2'
+    compile 'commons-cli:commons-cli:1.4'
+    compile 'info.picocli:picocli:4.4.0'
+    compile('com.google.guava:guava:27.1-jre') { force = true }
```
Comment (clstrfsck) [inline radixdlt-cli/build.gradle:49]: Why do we need a "force" here?  Would prefer if we didn't.

File: radixdlt-cli/src/main/groovy/com/radixdlt/cli/ValidatorKeyGenerator.groovy [hunk 2]
```diff
@@ -0,0 +1,49 @@
+package com.radixdlt.cli
+
+import com.radixdlt.crypto.ECKeyPair
+import com.radixdlt.crypto.RadixKeyStore
+import picocli.CommandLine
+
+@CommandLine.Command(name = "generate-validator-key",mixinStandardHelpOptions = true,
```
Comment (clstrfsck) [inline radixdlt-cli/src/main/groovy/com/radixdlt/cli/ValidatorKeyGenerator.groovy:18]: Would be better if we included some Javadoc here about how to run the command line applet.

**Review by clstrfsck — APPROVED** [review]

File: radixdlt-cli/src/main/groovy/com/radixdlt/cli/ValidatorKeyGenerator.groovy (hunk 2 above)
Comment (clstrfsck) [reply radixdlt-cli/src/main/groovy/com/radixdlt/cli/ValidatorKeyGenerator.groovy:18]: Also might be worth printing out the public key once complete.  Maybe replace the "Done" message with something like "Key AAkasjdfl3h4/12 written to keystore.ks", or changing the initial message to "Writing key asdfjhasdfl into new keystore keystore.ks".

File: radixdlt-cli/build.gradle (hunk 1 above)
Comment (siy) [reply radixdlt-cli/build.gradle:49]: after updating groovy there were conflicts so I was ought to force version to resolve them

Comment (siy) [reply radixdlt-cli/build.gradle:49]: I'll try to figure out how to resolve conflicts in less intrusive way

File: radixdlt-cli/src/main/groovy/com/radixdlt/cli/ValidatorKeyGenerator.groovy (hunk 2 above)
Comment (siy) [reply radixdlt-cli/src/main/groovy/com/radixdlt/cli/ValidatorKeyGenerator.groovy:18]: OK

**Review by clstrfsck — APPROVED** [review]
Looks good to me.

**Pull request merged**